lib/components/user/saved-trip-wizard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import SequentialPaneDisplay from './sequential-pane-display'
+
+/**
+ * This component is the new account wizard.
```

Comment:
A lot of the comments on components in this PR need work. This one should read something like: "This component shows a step-by-step progression of screens to build a MonitoredTrip."